nucliadb_models/nucliadb_models/search.py

```diff
@@ -341,16 +341,43 @@ def from_message(cls: Type[_T], message: PBShards) -> _T:
         return cls(**as_dict)
 
 
+class ParamDefaults:
+    query = dict(default="", title="Query", description="The query to search for")
+    advanced_query = dict(
+        default=None,
+        title="Advanced query",
+        description="An advanced query to search for. Follows the tantivy query parser syntax: https://docs.rs/tantivy/latest/tantivy/query/struct.QueryParser.html",  # noqa: E501
+    )
+    fields = dict(
+        default=[], title="Fields", description="The list of fields to search in"
```

Comment:
Can we mention how to use this field?

`f` -> only file fields
`f/file` -> only file field named `file`

but `t`, etc are supported as well right?

Reply:
done!